std-bits/base/src/main/java/org/enso/base/time/Date_Period_Utils.java

```diff
@@ -5,6 +5,20 @@
 import java.time.temporal.*;
 
 public class Date_Period_Utils implements TimeUtilsBase {
+  private static final long NANOSECONDS_IN_DAY = 86_400_000_000_000L;
```

Comment:
minor nitpick: Is it worth writing this as `24 * 60 * 60 * 1000_000_000`? The JVM will fold the constants anyway but it seems easier to see that the value is correct.

Reply:
Why does no one know its 86,400s in a day :)

Reply:
Yeah, maybe I will start to remember it in some time :D